libnfc/CMakeLists.txt

```diff
@@ -26,23 +29,23 @@ IF(UART_REQUIRED)
 ENDIF(UART_REQUIRED)
 
 IF(I2C_REQUIRED)
-  IF(WIN32)
-    # Windows is not supported at the moment
+  IF(NOT UNIX OR APPLE)
+    # Only Linux is supported at the moment
     #LIST(APPEND BUSES_SOURCES ../contrib/win32/libnfc/buses/i2c)
-    MESSAGE( FATAL_ERROR "I2C not (yet) supported under Windows!" )
-  ELSE(WIN32)
+    MESSAGE( FATAL_ERROR "I2C is only (yet) supported in Linux!" )
+  ELSE(NOT UNIX OR APPLE)
     LIST(APPEND BUSES_SOURCES buses/i2c)
-  ENDIF(WIN32)
+  ENDIF(NOT UNIX OR APPLE)
 ENDIF(I2C_REQUIRED)
 
 IF(SPI_REQUIRED)
-  IF(WIN32)
-    # Windows is not supported at the moment
+  IF(NOT UNIX OR APPLE)
```

Comment:
May I suggest to reverse the test and branches, to ease reading ?
if (unix or apple) ... set ON... else... set OFF